src/core/annotation.js

```diff
@@ -1000,6 +1000,7 @@ class WidgetAnnotation extends Annotation {
     this._defaultAppearanceData = parseDefaultAppearance(
       data.defaultAppearance
     );
+    this.data.defaultAppearanceData = this._defaultAppearanceData;
```

Comment:
You can shorten this to `data.defaultAppearanceData = ...` since `data` is a local variable reference to `this.data` here (similar to the lines above). I also think we can then remove usage of `this._defaultApperanceData` and replace it with usage of `this.data.defaultAppearanceData` so it's only tracked once in the data field instead of also as a member variable here.

Since this is an API addition this patch gets the label `[api-minor]`, and I'm only thinking if there is perhaps a shoter/better name here from an API perspective: `fontData`/`fontInformation` maybe, or is the data in default appearances not necessarily related to fonts?

Reply:
The specification mentions, that there might also be a text transformation matri inside the default appearance stream. So I am not sure if this would fit to  `fontData/fontInformation` . With the current implementation of the defaultAppearance parser, I would agree to rename this property, but on in the long term, this might not be the best choice.

Reply:
Let's indeed keep it like this; thanks.